pkg/reconciler/v1alpha1/pipeline/tasks.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2018 The Knative Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pipeline
+
+import (
+	"fmt"
+
+	"github.com/knative/build-pipeline/pkg/apis/pipeline/v1alpha1"
+	listers "github.com/knative/build-pipeline/pkg/client/listers/pipeline/v1alpha1"
+)
+
+// GetTasks retrieves all Tasks instances which the pipeline p references, using
+// lister l. If it is unable to retrieve an instance of a references Task, it will return
+// an error, otherwise it returnsa map from the name of the Task in the Pipeline to the
```

Comment:
typo `returns a`

Reply:
thanks @vdemeester ! even my commit message is actually filled with typos... :sweat_smile: